### PR TITLE
Revert commits does not need signedoffby check

### PR DESCRIPTION
**Pull request by MichelleXiaoxiaoWang**
Signed-off-by: Xiaoxiao Wang <michellejavajs@gmail.com>

### CODE REVIEW

**Comment by MichelleXiaoxiaoWang** [comment]
#### **revert**
'''
screen shot 2018-07-25 at 9 29 49 am
'''

File: buildenv/jenkins/jobs/infrastructure/signedOffByCheck [hunk 1]
```diff
@@ -56,23 +56,28 @@ timeout(time: 6, unit: 'HOURS') {
...
-                                    echo "The following commit appears to have a proper sign-off:\n${COMMIT}" 
-                                    WRONG_COMMIT = false
-                                    break
-                                }
-                            }
-                            if ( WRONG_COMMIT ) {
-                                echo "FAILURE - The following commit appears to have an incorrect sign-off:\n${COMMIT}"
-                                BAD_COMMITS << "${COMMIT}"
-                                FAIL = true
+                            if (splitCommit[splitCommit.size() - 1] ==~ /.*This reverts commit.*/){
```
Comment (AdamBrousseau) [inline buildenv/jenkins/jobs/infrastructure/signedOffByCheck]: is `splitCommit[splitCommit.size() - 1]` the same as `splitCommit[-1]` ?

File: buildenv/jenkins/jobs/infrastructure/signedOffByCheck [hunk 2]
```diff
@@ -56,23 +56,28 @@ timeout(time: 6, unit: 'HOURS') {
                             def WRONG_COMMIT = true
                             def splitCommit = COMMIT.split("\\r?\\n")
                             def lastBlankLine = 0
-                            for (int i = splitCommit.size() - 1; i >= 0; i--) {
```
Comment (AdamBrousseau) [inline buildenv/jenkins/jobs/infrastructure/signedOffByCheck]: We can probably be a little more strict on this regex. SHAs are always 40 characters and only contain numbers and letters a-f. It also seems to end in a period. Is there a set number of spaces at the beginning of the line?
Perhaps something like:
`^\s{4}This reverts commit [0-9a-f]{40}\.$`

Comment (MichelleXiaoxiaoWang) [reply buildenv/jenkins/jobs/infrastructure/signedOffByCheck]: fixed

File: buildenv/jenkins/jobs/infrastructure/signedOffByCheck (hunk 1 above)
Comment (MichelleXiaoxiaoWang) [reply buildenv/jenkins/jobs/infrastructure/signedOffByCheck]: fixed

File: buildenv/jenkins/jobs/infrastructure/signedOffByCheck (hunk 2 above)
Comment (MichelleXiaoxiaoWang) [reply buildenv/jenkins/jobs/infrastructure/signedOffByCheck]: fixed

**Pull request merged**